README.md

```diff
@@ -29,4 +42,35 @@ sudo apt update
 sudo apt install lubuntu-desktop
 ```
 
-The system will need to be re-started with the Desktop environment in order to run the Workshop. 
\ No newline at end of file
+The system will need to be re-started with the Desktop environment in order to run the Workshop. 
+
+## Installation instructions on macOS:
+
+Tested on Mojave (10.14) using Python 3.7 installed directly from Python.org. Python installed with `homebrew` should also work, but there is a [known issue](https://github.com/matplotlib/matplotlib/issues/9637#issuecomment-515081488) when scrolling on a Matplotlib figure embedded in Tkinter that causes it to crash. Anaconda Python should also work - not tested, thought.
+
+```bash
+git clone https://github.com/ImperialCollegeLondon/python-guis.git
+cd python-guis
+python3 -mvenv venv
+. venv/bin/activate
+pip install -U setuptools wheel pip
+pip install -e .
+garden install matplotlib --kivy
+```
+
+## Installation instructions on Windows 10:
+
+The following installation instructions have been tested in Windows 10 using Git Bash (might require some extra configuration) and PowerShell (you might need to disable temporarily the security directives in order to activate the virtual environment) with Anaconda Python 3.7.
+
+```bash
+git clone https://github.com/ImperialCollegeLondon/python-guis.git
+cd python-guis
+python -mvenv venv
+. venv/Scripts/activate
+pip install -U setuptools wheel pip
+pip install docutils pygments pypiwin32 kivy_deps.sdl2 kivy_deps.glew
+pip install -e .
+garden install matplotlib --kivy
+```
+
+However, Kivy might not work as expected. There is an unresolved issue - system specific - where you might get a complain about having an old version of OpenGL or just a black window with no widgets. If that's your case, try one of the solutions suggested [here](https://stackoverflow.com/questions/52109670/cannot-display-anything-but-a-solid-color-window-with-kivy).
```

Comment:
Is there a more official link to a Kivy issue that could go here? None of the proposed solutions seemed to work for that person.

Reply:
That's the problem. As it is not clear when this happens - and who is to blame for it - there is no official response. The issues in the Kivy GitHub related to this have been closed because they cannot reproduce it. I've included this link because it mentions all the possible solutions that I have found scattered somewhere else, even though none of them work for this person.